layouts/shortcodes/cloud-intro.html

```diff
@@ -0,0 +1,12 @@
+<div>
+    <p>
+    Pulumi's infrastructure-as-code SDK helps you create, deploy, and manage {{ .Get 0 }} containers,
+    serverless functions, and infrastructure using real programming languages. The Pulumi
+    <a href="{{ .Get 1 }}" target="_blank">{{ .Get 0 }} provider package</a> and
```

Comment:
I'm wondering if we should cut this last sentence. This is the first step of the guide and we're already linking elsewhere? I'm wondering if we should avoid links here to encourage moving forward in the guide.